### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by kbarkhqs**

### CODE REVIEW

File: documentation/src/physical_types/spins.md [hunk 1]
```diff
@@ -168,27 +168,6 @@ hamiltonian.add_operator_product(pp, 1.0.into()).unwrap();
...
-operator
-    .add_operator_product(pp, CalculatorComplex::new(1.0, 1.5))
-    .unwrap();
-operator
-    .add_operator_product(PauliProduct::new().z(3), CalculatorComplex::new(1.0, 1.5))
-    .unwrap();
-println!("{}", operator);
-```
-
 The equivalent code in python:
```
Comment (dberthault) [inline documentation/src/physical_types/spins.md:171]: This line doesn't make much sense with the rust example removed

**Review by dberthault — CHANGES_REQUESTED** [review]
Shouldn't similar changed be also done in struqture 1.x ?

File: documentation/src/physical_types/spins.md (hunk 1 above)
Comment (kbarkhqs) [reply documentation/src/physical_types/spins.md:171]: The Rust example (and the title) was duplicated, if you open the lines a little further up, you'll see that it's present there too

Comment (dberthault) [reply documentation/src/physical_types/spins.md:171]: Yes but the rust example is more complete that the python one so they're not really equivalent, I think we should add the operator.set and hamiltonian part to the python example

Comment (kbarkhqs) [reply documentation/src/physical_types/spins.md:171]: Ah good point! I'll add this.

Comment (kbarkhqs) [reply documentation/src/physical_types/spins.md:171]: I have updated the python code snippet

**Review by dberthault — APPROVED** [review]

**Pull request merged**